lib/server/routesHandlers/httpRequestHandler.ts

```diff
@@ -28,6 +28,7 @@ export const overloadHttpRequestWithConnectionDetailsMiddleware = async (
       localHostname.match(regex)
     ) {
       const url = new URL(`http://${req.hostname}${req.url}`);
+      url.hostname = req.hostname.replace(/-[0-9]{1,2}/, '');
```

Comment:
@aarlaud, just to be sure I'm getting the replacement logic: if the hostname is `my-server-123-name` => it will be `my-server-223-name`?